src/libslic3r/PrintConfig.cpp

```diff
@@ -148,7 +148,14 @@ void PrintConfigDef::init_common_params()
                    "the API Key or the password required for authentication.");
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionString(""));
-
+    
+    def = this->add("printhost_slug", coString);
+    def->label = L("Printer");
+    def->tooltip = L("Name of the printer");
+    def->gui_type = "select_open";
```

Comment:
this line def->gui_type = "select_open"; makes the slicer crash.  It's unsupported and maybe un-needed (print_host use nothing).